azkaban-common/src/test/java/azkaban/executor/JdbcExecutorLoaderTest.java

```diff
@@ -293,6 +334,209 @@ public void testUploadExecutableNode() throws Exception {
 
   }
 
+  /* Test active executors fetch from empty executors */
+  @Test
+  public void testFetchEmptyExecutors() throws Exception {
+    if (!isTestSetup()) {
+      return;
```

Comment:
better to throw an exception, otherwise this test will silently pass every time the setup step is not properly done.